src/Features/CSharp/Portable/AddAccessibilityModifiers/CSharpAddAccessibilityModifiersDiagnosticAnalyzer.cs

```diff
@@ -131,12 +133,32 @@ public CSharpAddAccessibilityModifiersDiagnosticAnalyzer()
 
             // Have an issue to flag, either add or remove. Report issue to user.
             var additionalLocations = ImmutableArray.Create(member.GetLocation());
+
+            var properties = ImmutableDictionary.CreateBuilder<string, string>();
+            var ruleName = CodeStyleOptions.RequireAccessibilityModifiers.StorageLocations.OfType<EditorConfigStorageLocation<CodeStyleOption<AccessibilityModifiersRequired>>>().FirstOrDefault();
```

Comment:
I really dislike all of this.  It makes things so leaky.  

Ths is similar to my issue with how we did codecleanup in the first place.  Instead of keeping concerns separated, we made it so that different systems have a deep and leaky knowledge of how other parts of the ysstem work.  This is extremely brittle and difficult to maintain.